res/database/Default/user/scr0llbaer.xml

```diff
@@ -467,30 +510,121 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="9f1cacc4-aa56-4f4a-9a9f-8c21c4bd3a7d" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="56bcb1ff-8a5d-4d91-b3f7-ad6958b3855b" creator="9551">
+			<title>
+				<de>%recreational%%athletes% aus %randomcity% und anderswo</de>
```

Comment:
You could even add variations to "anderswo"

anderswo
Umgebung
Laufnähe (as pun for athletes :D)

You might even "prepend" stuff before %randomcity%
(nichts)
Sportstadt
Talentschmiede
...

Reply:
I had thought about "Umgebung", but the idea is actually that the `%randomcity%` should serve more as an example, and not that it's about sports from that city/region only.

> You might even "prepend" stuff

Then it gets too long I think.